models/setup_for_test.go

```diff
@@ -30,8 +30,8 @@ func TestMain(m *testing.M) {
 	setting.RunUser = "runuser"
 	setting.SSH.Port = 3000
 	setting.SSH.Domain = "try.gitea.io"
-	setting.RepoRootPath = "/repos"
-	setting.AppDataPath = "/appdata"
+	setting.RepoRootPath = "/tmp/repos"
```

Comment:
@ethantkoenig could you send another PR to fix this so that we don't need to revert this one.